src/IlluminaSampleSheet/V1/DataInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\IlluminaSampleSheet\V1;
+
+use MLL\Utils\CSVArray;
+
+/**
+ * @phpstan-import-type CSVPrimitive from CSVArray
+ */
+interface DataInterface
```

Comment:
Soll das auch ein Interface fuer alle Versionen sein? Dann wuerde ich das auslagern und nicht im V1 namespace haben. Aber das war wahrscheinlich auch erst einmal ein Versuch?

Reply:
Ich denke, dass das (in der aktuellen Art) weg kann wenn wir eine DataSection pro sampleSheet-Typen haben. Da `IlluminaSampleSheet/V2/BclConvert/DataSection.php` so speziell ist, benötigen wir glaube ich auch keine auf Root level `IlluminaSampleSheet/`.